_posts/2017-10-18-understanding-colourblindness.md

```diff
@@ -18,7 +18,7 @@ I've avoided going into the different types here, as the principles are the same
 
 ## What does this mean for accessibility on the web?
 
-Don't use on colour alone to communicate. As a result, you don't risk making life difficult for colour blind users. 
+Don't rely on colour alone to communicate. As a result, you don't risk making life difficult for colour blind users. For example, hyperlinks are underlined and coloured.
```

Comment:
This would read better as: 

> …you avoid making life difficult…

Something about `don't risk` feels awkward.

Not something you've changed in this PR, but perhaps while it's open?

Reply:
I don't agree, I'm afraid. I spent a while thinking about this when drafting. 

I think `don't risk` feels most literal. Let me try and explain.

This comes down to the fact that there are many types of colourblindness, and most people with some form of colourblindness can tell the difference between many colours. 

If this were to read `you avoid making life difficult`, that gives the impression that if you get your colours wrong, it affects everyone with different colour vision. If this were the case, and one person with a colour vision impairment could see it correctly, it would suggest that there's actually nothing wrong with the colour choice.

On the other hand, if we say that you _risk_ making it difficult to understand, rather than implying you'll definitely make it difficult to understand, we make it clear that you can't be sure if you have an issue based on one person's experiences. 

tl;dr version: I was concerned that this suggestion sounded too binary, rather than reflecting the spectrum of different types of colour vision. 

If you have a suggestion for a way to make this clear in as short and simple a sentence, I'd be keen to hear it!

Reply:
I actually agree with the awkwardness of "don't risk", but also agree with your reasoning for trying to convey the idea this way. 

I think instead of "don't risk" it should be "won't risk" as that is a more natural way to chain the sentences together. Cause and effect really.  Right now it reads as two commands, but the second is meant to be the outcome of the first "don't".  e.g. "don't do this so this won't happen."  Not "don't do this so this don't happen."

"Don't rely on colour alone to communicate. As a result, you *won't* risk making life difficult for colour blind users."

On a different note, "making life difficult" may be a bit hyperbolic, as it's really a difficulty in understanding their present intake of content. Their "life" (hopefully) hasn't gone down the tubes because the "username" label of a sign in field doesn't meet WCAG color contrast :)